Composer/packages/client/src/pages/botProject/SkillHostEndPoint.tsx

```diff
@@ -87,15 +95,10 @@ export const SkillHostEndPoint: React.FC<SkillHostEndPointProps> = (props) => {
       <TextField
         ariaLabel={formatMessage('Skill host endpoint url')}
         data-testid={'SkillHostEndPointTextField'}
+        defaultValue={skillHostEndpoint}
         label={formatMessage('Skill host endpoint URL')}
         placeholder={formatMessage('Enter Skill host endpoint URL')}
-        value={skillHostEndpoint}
-        onChange={(e, value) => {
```

Comment:
Why does the jump in cursor position occur @alanlong9278?  Typically we debounce something only when a intensive operation is performed for the keystroke. It doesn't look to be that case here? If you dont want to update the recoil state immediately having a local state and then persisting to the recoil state onBlur seems better.

Reply:
1. Cursor position error is because of recoil value change, so i set default value. 
2. Add debounce is for reducing api call. `setSettings` will trigger updating appSetting.json api call.

Reply:
Use `state` & `onBlur` to call api to align with other setting property changes